src/main/scala/viper/gobra/frontend/Config.scala

```diff
@@ -359,8 +359,30 @@ class ScallopGobraConfig(arguments: Seq[String], isInputOptional: Boolean = fals
   )
 
   banner(
-    s""" Usage: ${GoVerifier.name} -i <input-files> [OPTIONS] OR
-       |  ${GoVerifier.name} -p <directory> [OPTIONS]
+    s""" ${GoVerifier.name} supports three modes for specifying input files that should be verified:
+       |  ${GoVerifier.name} -i <input-files> [OPTIONS] OR
+       |  ${GoVerifier.name} -p <directories> <optional project root> [OPTIONS] OR
+       |  ${GoVerifier.name} -r <optional project root> <optional include and exclude package names> [OPTIONS]
+       |
+       | Mode 1 (-i):
+       |  The first way interprets the list of files as one package that should be verified.
```

Comment:
If I understand correctly, then -i complains if it gets files belonging to different packages. Maybe something like "the first mode takes a list of files. Every file must belong to the same package. Missing package files are not considered for type checking and verification.".

Reply:
```suggestion
       |  The first mode takes a list of files, which must belong to the same package.
       |  Files belonging to the same package but missing in the list are not considered for type-checking and verification.
```

Reply:
I would rather write "The first mode takes a list of files that must belong to the same package."